### PR TITLE
Adds 2 Chef job xp rewards

### PR DESCRIPTION
**Pull request by Carbadox**
## About the PR 
Adds unique chef mitts and "kiss the cook" apron to chef job xp rewards
Also lowered spectroscopic monocle to level 0 because i want more people to use it.

## Why's this needed? 
To expand the job rewards system

## Changelog

```changelog
(u)Carbadox
(*)Added 2 new Chef job rewards. Go check 'em out.
```

### CODE REVIEW

File: code/WorkInProgress/JobXPRewards.dm [hunk 1]
```diff
@@ -617,3 +617,42 @@ mob/verb/checkrewards()
...
+/datum/jobXpReward/chef/ovenmitts
+	name = "Chef's Oven Mitts"
+	desc = "Finally, after years of burning your fingers on baking trays, soup pots and deepfryers, you get your own pair of oven mitts! "
+	required_levels = list("Chef"=0)
+	icon_state = "?"
+	claimable = 1
+	claimPerRound = 1
+
+	activate(var/client/C)
+		boutput(C, "You fumble about in your pokcets and managed to find your oven mitts.")
```
Comment (jan-antilles) [inline code/WorkInProgress/JobXPRewards.dm:632]: Quick typo: it's "pockets" not "pokcets"!

**Comment by DimWhat** [comment]
image
pls add this monkey version with the rest of the PR

File: code/WorkInProgress/JobXPRewards.dm [hunk 2]
```diff
@@ -547,7 +547,7 @@ mob/verb/checkrewards()
 /datum/jobXpReward/bartender/spectromonocle
 	name = "Spectroscopic Monocle"
 	desc = "Now you can look dapper and know which drinks you poisoned at the same time"
-	required_levels = list("Bartender"=5)
+	required_levels = list("Bartender"=0)
```
Comment (mordent-goonstation) [inline code/WorkInProgress/JobXPRewards.dm:550]: Not part of the changelist. Suspect changed as part of debugging?

File: code/WorkInProgress/JobXPRewards.dm [hunk 3]
```diff
@@ -617,3 +617,42 @@ mob/verb/checkrewards()
...
+	desc = "Finally, after years of burning your fingers on baking trays, soup pots and deepfryers, you get your own pair of oven mitts! "
+	required_levels = list("Chef"=0)
+	icon_state = "?"
+	claimable = 1
+	claimPerRound = 1
+
+	activate(var/client/C)
+		boutput(C, "You fumble about in your pockets and managed to find your oven mitts.")
+		var/obj/item/I = new/obj/item/clothing/gloves/black/ovenmitts()
+		I.set_loc(get_turf(C.mob))
```
Comment (mordent-goonstation) [inline code/WorkInProgress/JobXPRewards.dm:634]: Check this works, but I don't see why you couldn't do as one thing.
```suggestion
		var/obj/item/I = new/obj/item/clothing/gloves/black/ovenmitts(get_turf(C.mob))
```

File: code/WorkInProgress/JobXPRewards.dm [hunk 4]
```diff
@@ -617,3 +617,42 @@ mob/verb/checkrewards()
...
+		C.mob.put_in_hand(I)
+		return
+
+/datum/jobXpReward/chef/kissthecook
+	name = "Tacky 'Kiss The Cook' apron"
+	desc = "It gets lonely in the kitchen and frankly, neither the bartender or the monkeys are particularly good snoggers"
```
Comment (mordent-goonstation) [inline code/WorkInProgress/JobXPRewards.dm:640]:
```suggestion
	desc = "It gets lonely in the kitchen and, frankly, neither the bartender or the monkeys are particularly good snoggers"
```

File: code/WorkInProgress/JobXPRewards.dm [hunk 5]
```diff
@@ -617,3 +617,42 @@ mob/verb/checkrewards()
...
+	activate(var/client/C)
+		var/apron = C.mob.find_type_in_hand(/obj/item/clothing/suit/apron/)
+
+		if(!(apron))
+			boutput(C.mob, "You need to be in possession of an apron to claim this item")
+			return
+		C.mob.remove_item(apron)
+		qdel(apron)
+		var/obj/item/I = new path_to_spawn()
+		I.set_loc(get_turf(C.mob))
```
Comment (mordent-goonstation) [inline code/WorkInProgress/JobXPRewards.dm:655]:
```suggestion
		var/obj/item/I = new path_to_spawn(get_turf(C.mob))
```

File: code/obj/item/clothing/gloves.dm [hunk 6]
```diff
@@ -239,6 +239,15 @@ ABSTRACT_TYPE(/obj/item/clothing/gloves)
 		qdel(src)
 		user.put_in_hand_or_drop(new /obj/item/clothing/gloves/fingerless)
 	else . = ..()
+
+/obj/item/clothing/gloves/black/ovenmitts
```
Comment (mordent-goonstation) [inline code/obj/item/clothing/gloves.dm:243]: Dislike `/gloves/black` and them being not that.

File: code/WorkInProgress/JobXPRewards.dm [hunk 7]
```diff
@@ -617,3 +617,42 @@ mob/verb/checkrewards()
 		I.set_loc(get_turf(C.mob))
 		C.mob.put_in_hand(I)
 		return
+
+/////////////Chef////////////////
+
+/datum/jobXpReward/chef/ovenmitts
+	name = "Chef's Oven Mitts"
+	desc = "Finally, after years of burning your fingers on baking trays, soup pots and deepfryers, you get your own pair of oven mitts! "
+	required_levels = list("Chef"=0)
```
Comment (mordent-goonstation) [inline code/WorkInProgress/JobXPRewards.dm:626]: If it's a zero XP reward, why not just put in their spawn inventory?

File: code/WorkInProgress/JobXPRewards.dm (hunk 2 above)
Comment (Carbadox) [reply code/WorkInProgress/JobXPRewards.dm:550]: I listed it in About this PR. I'll add it to the changelog

File: code/WorkInProgress/JobXPRewards.dm (hunk 7 above)
Comment (Carbadox) [reply code/WorkInProgress/JobXPRewards.dm:626]: The main goal is to just provide basic Job XP rewards, to add some meat to the bare bones system, so to speak. I'd prefer not to clog player inventory's and it makes the items feel more special to players.

File: code/obj/item/clothing/gloves.dm (hunk 6 above)
Comment (Carbadox) [reply code/obj/item/clothing/gloves.dm:243]: Not sure what that means. It's a child, so it's just going to inherit the gloves heat protection. It uses a unique name, description and glove fibre

**Pull request closed without merging**